pkg/resources/resources.go

```diff
@@ -471,9 +471,9 @@ const (
 	EtcdTLSKeySecretKey = "etcd-tls.key"
 
 	// EtcdRestoreS3CredentialsSecret names the secret expected in seed kube-system that must contain S3 credentials for etcd backup restores
-	EtcdRestoreS3CredentialsSecret     = "s3-credentials"
-	EtcdRestoreS3AccessKeyIDKey        = "ACCESS_KEY_ID"
-	EtcdRestoreS3SecretKeyAccessKeyKey = "SECRET_ACCESS_KEY"
+	EtcdRestoreS3CredentialsSecret              = "s3"
```

Comment:
While we're at it, it might be the perfect time to put the purpose into the secret name? Maybe call it `backup-credentials`?

Reply:
The thing is we are gonna have multiple storage providers so not only s3. Thus in the future we will have, s3, google-file storage and azure storage. That's why we agreed on regarding naming it.

Reply:
`backup-s3` then maybe? I can see your point, but we recently had a case where our infra had a "bucket-reader" account that was used by an unknown number of components and it became much easier to reason about once we renamed the account to "prow-deck". The current confusion as to who and what should use the secret should be example enough why we should include "backup" somewhere in the name. :)

Another point: What if another component also needs S3 credentials, but for something totally different? Would we then want to have "s3-2"? "s3-second"? ^^

Reply:
`backup-s3` sounds good to me